library/tasks/HISAT2.wdl

```diff
@@ -0,0 +1,313 @@
+task HISAT2PairedEnd {
+  File hisat2_ref
+  File fq1
```

Comment:
I think here and below we would want `fastq1 fastq2` vs `fq1`

Reply:
good catch!